### PR TITLE
Feat/finished up 3 4 5 screen

### PR DESCRIPTION
**Pull request by ruslan-milinchuk**

### CODE REVIEW

File: css/beautiful_interface.css [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+}
+
+.interface-screen__section-interface-img:first-child {
+    margin-left: -70px;
+}
+.interface-screen__section-interface-img:last-child {
+    margin-right: -70px;
+}
+
+.interface-screen-logo-NewProvidence {
```
Comment (andriy-ilin) [inline css/beautiful_interface.css]: interface-screen-logo-NewProvidence -> interface-screen__logo

File: css/day_protected.css [hunk 2]
```diff
@@ -0,0 +1,58 @@
+.bgIphoneProtected {
```
Comment (andriy-ilin) [inline css/day_protected.css]: change class name .bgIphoneProtected

File: css/day_protected.css [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    color: rgb(38, 39, 45);
+    font-family: "Varela Round";
+    font-size: 42px;
+    font-weight: 400;
+    line-height: 52px;
+    width: 60%;
+    margin-bottom: 30px;
+}
+
+.day-protected-screen-description {
```
Comment (andriy-ilin) [inline css/day_protected.css]: day-protected-screen-description -> day-protected-screen__description

File: css/day_protected.css [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    background: url("../img/iphone.6.protected.png") no-repeat;
+    background-position: left -30% top 0% ;
+}
+
+.day-protected-screen {
+    margin: 80px 10%  230px 63% ;
+
+}
+
+.day-protected-screen-title {
```
Comment (andriy-ilin) [inline css/day_protected.css]: day-protected-screen-title -> day-protected-screen__title

divider between block and element is `__`

File: css/main_screen.css [hunk 5]
```diff
@@ -0,0 +1,109 @@
+.header{
```
Comment (andriy-ilin) [inline css/main_screen.css]: header block to new file css with name `header.css`

File: css/main_screen.css [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+
+.header__section-item-left, .header__section-item-right {
+    margin-right: 30px;
+}
+
+.header__section-item-right:last-child {
+    margin-right: 0;
+}
+
+.header__section-linc {
```
Comment (andriy-ilin) [inline css/main_screen.css]: linc -> link

File: css/main_screen.css [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+    text-decoration: none;
+    color: rgb(38, 39, 45);
+    font-size: 16px;
+}
+
+.header__section-linc:hover {
```
Comment (andriy-ilin) [inline css/main_screen.css]: linc -> link

File: css/main_screen.css [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+}
+
+.main-screen__content-title {
+    color: rgb(38, 39, 45);
+    font-size: 26px;
+    font-weight: 300;
+    line-height: 34px;
+}
+
+.divider {
```
Comment (andriy-ilin) [inline css/main_screen.css]: divider class to file `style.css`

File: css/style.css [hunk 9]
```diff
@@ -0,0 +1,31 @@
...
+    padding: 0;
+}
+body {
+    margin: 0;
+}
+.screen {
+    width: 100%;
+    margin-bottom: 150px;
+}
+.bgNature {
```
Comment (andriy-ilin) [inline css/style.css]: rename

Comment (andriy-ilin) [reply css/style.css]: create class `main-screen` and use this props `background: url("../img/bgNature.png") no-repeat;
    background-size: cover;`
remove it to `main-screen.css`

File: css/style.css [hunk 10]
```diff
@@ -0,0 +1,31 @@
...
+    background: url("../img/bgNature.png") no-repeat;
+    background-size: cover;
+}
+
+.screen--medium {
```
Comment (andriy-ilin) [inline css/style.css]: rename `.screen--medium` maybe `screen` -? no-? have you large screen or small screen -?

File: css/day_protected.css [hunk 11]
```diff
@@ -0,0 +1,58 @@
...
+    margin-top: 130px;
+}
+
+.day-protected-screen__btn-protected {
+    margin-right: 20px;
+    padding: 0;
+    display: flex;
+    justify-content: space-around;
+    align-items: center;
+    width: 202px;
```
Comment (andriy-ilin) [inline css/day_protected.css]: width and height not good in this case
use padding it better

File: css/day_protected.css [hunk 12]
```diff
@@ -0,0 +1,58 @@
...
+    background-color: rgb(38, 125, 244);
+    color: white;
+}
+
+.day-protected-screen__btn-info {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    width: 142px;
+    height: 42px;
```
Comment (andriy-ilin) [inline css/day_protected.css]: width and height not good in this case
use padding it better

change in all btn

File: css/we_everywhere.css [hunk 13]
```diff
@@ -0,0 +1,45 @@
+.we-everywhere-screen-bg-world-map {
```
Comment (andriy-ilin) [inline css/we_everywhere.css]: divider between block and element is `__` change all in file.
we-everywhere-screen-bg-world-map -> we-everywhere-screen__bg-world-map

Comment (andriy-ilin) [reply css/we_everywhere.css]: why you don't use just `we-everywhere-screen` without `bg-world-map` -?
it's block, change

File: css/we_everywhere.css [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+    background: url("../img/World_Maps.png") no-repeat;
+    background-position: right -30% bottom 50%;
+
+}
+
+.we-everywhere-screen {
+    margin: 80px 63%  230px 0%
+}
+
+.we-everywhere-screen-title {
```
Comment (andriy-ilin) [inline css/we_everywhere.css]: divider bettween block and element `__`

File: css/we_everywhere.css [hunk 15]
```diff
@@ -0,0 +1,45 @@
...
+    color: rgb(38, 39, 45);
+    font-family: "Varela Round";
+    font-size: 42px;
+    font-weight: 400;
+    line-height: 52px;
+    width: 60%;
+    margin-bottom: 30px;
+}
+
+.we-everywhere-screen-description {
```
Comment (andriy-ilin) [inline css/we_everywhere.css]: divider bettween block and element __

File: css/we_everywhere.css [hunk 16]
```diff
@@ -0,0 +1,45 @@
...
+    font-size: 20px;
+    line-height: 28px;
+    margin-right: 20%;
+    margin-bottom: 30px;
+
+}
+.we-everywhere-screen-partners {
```
Comment (andriy-ilin) [inline css/we_everywhere.css]: divider bettween block and element __

File: css/we_everywhere.css [hunk 17]
```diff
@@ -0,0 +1,45 @@
...
+    font-family: Lato;
+    font-size: 12px;
+    font-weight: 700;
+    line-height: 22px;
+    text-transform: uppercase;
+    letter-spacing: 1.2px;
+    margin-top: 60px;
+}
+
+.we-everywhere-screen-partners-logo-dropbox {
```
Comment (andriy-ilin) [inline css/we_everywhere.css]: divider bettween block and element __

File: css/we_everywhere.css [hunk 18]
```diff
@@ -0,0 +1,45 @@
...
+    margin-right: 25px;
+}
+.we-everywhere-screen-partners-logo {
```
Comment (andriy-ilin) [inline css/we_everywhere.css]: divider bettween block and element __

File: css/day_protected.css (hunk 2 above)
Comment (ruslan-milinchuk) [reply css/day_protected.css]: can be .bg-protected?

**Pull request merged**